.ci/Jenkinsfile

```diff
@@ -111,7 +111,7 @@ pipeline {
                   unstash 'source'
                   withGoEnv(version: "${GO_VERSION}"){
                     dir(BASE_DIR){
-                      sh script: '.ci/scripts/install-dependencies.sh', label: 'Install dependencies'
+                      retryWithSleep(retries: 2, seconds: 5, backoff: true){ sh script: '.ci/scripts/install-dependencies.sh', label: 'Install dependencies' }
```

Comment:
Add some CI resilience when accessing third-party systems to download dependencies are temporarily unavailable